tests/chart_tests/test_elasticsearch.py

```diff
@@ -650,3 +650,34 @@ def test_elasticsearch_nginx_deployment_overrides(self, kube_version):
         assert len(docs) == 1
         c_by_name = get_containers_by_name(docs[0])
         assert c_by_name["nginx"]["securityContext"] == {"runAsNonRoot": True}
+
```

Comment:
Elasticsearch is missing the `"persistentVolumeClaimRetentionPolicy" not in doc["spec"]` test case.